notebooks/0201_create-composite-history.py

```diff
@@ -60,12 +60,14 @@
 class CEDSOption(StrEnum):
     """CEDS options"""
 
+    Drive_2025_03_18 = auto()
```

Comment:
```suggestion
    Zenodo_2025_03_18 = auto()
```

We got this from Zenodo, no?